go.mod

```diff
@@ -249,7 +220,7 @@ replace (
 	k8s.io/kubectl => k8s.io/kubectl v0.17.1
 	k8s.io/kubelet => k8s.io/kubelet v0.17.1
 
-	k8s.io/kubernetes => github.com/kasturinarra/kubernetes v1.17.0-kcpqe
+	k8s.io/kubernetes => github.com/Phaow/kubernetes v0.6.2-kcp
```

Comment:
@Phaow any reason for this modification? Did we add/remove more code in here ?

Reply:
Yeah. I did 2 enchance commits as your basic version. I could file a PR to your repo for review if needed.
**[Remove unnecessary check and test for kcp](https://github.com/Phaow/kubernetes/commit/1bdac27696548e0adea4cf4c74a95d69ed5061ef)**
**[Enchance log output](https://github.com/Phaow/kubernetes/commit/21e424c777ff666d475af60e0ff444dd8f85ed3f)**

Reply:
@Phaow not really necessary, that is fine, thank you !!

Reply:
Hi @kasturinarra ,TBH, The better way is we add the change to our kcp folked [kubernetes repo](https://github.com/kcp-dev/kubernetes) to tag a version(a specific branch). Seems need our DEV's help. But we could enchance it better to avoid file too many PR to the repo.

Reply:
@Phaow if we needed to do that, let us do it that way, not sure who are the dev's you are referring here, if kcp dev i can check with andy if not, then let us check with jianzhang on who is the dev person we need to talk to with respect to kubernetes, thanks !!

Reply:
Yeah, I mean our kcp DEV andy, it's not a must step. Just a better way to manage the go modules. Current way is also ok for us.